### DISCUSSION TITLE
Internal server error 500 on creating mail server

### DISCUSSION THREAD

**Discussion by Brutus5000** [postalserver/postal#1845] [Help with using Postal] [answered]
I have setup Postal on kubernetes. All workers are running. The database is initialized. The rabbitmq vhost is setup and and a rabbimq queue `deliver-jobs-main` was created. A user is setup.

I could succesfully create a domain.
However when adding a server and clicking on build server I get a web response of 500 internal server error with no additional data.
Looking at all logs (logs to stdout is enabled) all I can see are the http calls

```
10.42.0.1 - - [07/Feb/2022:16:23:01 +0000] "GET /org/faf/servers/new HTTP/1.1" 200 - 0.0493
10.42.0.1 - - [07/Feb/2022:16:23:09 +0000] "GET /assets/button-spinner-positive-cf834a659fbdd516aa664cd584aa33efc0bddba08a025b488cf0b7d1ef32e487.gif HTTP/1.1" 200 404 0.0105
10.42.0.1 - - [07/Feb/2022:16:23:09 +0000] "POST /500 HTTP/1.1" 500 46 0.0652
```

Can I increase the log level somehow to get more details?

**Comment by willpower232** [comment, marked as answer]
Unfortunately you will need to go onto the running pod to see its log files, the stdout logs are from the running process which is the web server.

**Reply by Brutus5000** [reply]
Thanks alot for the fast answer, I found an issue with the database setup (the user is not allowed to create new databases).
For people reading this, the "default" log is under `/app/log/production.log`

**Comment by RickoT** [comment]
I have resolved this by adding appropriate privileges to a wildcard database name for the user I use for postal.  I don't want to grant the postal user with full privileges so I used this to do that.  Now postal will be able to access the postal database, as well as create and manage the postal server databases, without granting root access to mysql to the postal user.  I just set this up today, so far it seems to be working out

`GRANT ALL PRIVILEGES ON 'dbprefix%'.* TO 'postaluser'@'%' IDENTIFIED BY 'postalpass';`

**Reply by willpower232** [reply]
That is correct yes. As per the documentation

> If you are unable or unwilling to grant root access, the database user you create separately needs all permissions on databases called postal and postal-* (this prefix can be configured in the message_db part of your configuration).
